package.json

```diff
@@ -44,36 +44,36 @@
     "@babel/preset-env": "7.21.4",
     "@babel/preset-react": "7.18.6",
     "@babel/preset-typescript": "7.21.4",
-    "@mui/material": "^5.12.2",
     "@mui/styled-engine-sc": "^5.12.0",
-    "@mui/styles": "^5.12.0",
-    "@nrwl/cli": "13.10.2",
-    "@nrwl/cypress": "13.10.2",
-    "@nrwl/eslint-plugin-nx": "13.10.2",
-    "@nrwl/express": "13.10.2",
-    "@nrwl/jest": "13.10.2",
-    "@nrwl/linter": "13.10.2",
-    "@nrwl/node": "13.10.2",
-    "@nrwl/react": "13.10.2",
-    "@nrwl/tao": "13.10.2",
-    "@nrwl/web": "13.10.2",
-    "@nrwl/workspace": "13.10.2",
-    "@testing-library/react": "12.1.5",
-    "@types/express": "4.17.13",
+    "@mui/material": "^5.12.2",
+    "@nrwl/js": "16.0.3",
+    "@nrwl/tao": "16.0.3",
+    "@nx/cypress": "16.0.3",
+    "@nx/eslint-plugin": "16.0.3",
+    "@nx/express": "16.0.3",
+    "@nx/jest": "16.0.3",
+    "@nx/linter": "16.0.3",
+    "@nx/node": "16.0.3",
+    "@nx/webpack": "16.0.3",
+    "@nx/workspace": "16.0.3",
+    "@nx/react": "16.0.3",
+    "@testing-library/react": "13.4.0",
+    "@types/dotenv": "^8.2.0",
```

Comment:
You have fixed your previous bad conflict resolutions during a rebase, when a bunch of packages were reverting changes from #373. Now, you only have one bad conflict resolution, because you've brought back `@types/dotenv` which got removed in #376.